src/app/components/card/card.components.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'app-card',
+  templateUrl: './card.component.html',
+  styleUrls: ['./card.component.scss'],
+})
+export class CardComponent implements OnInit {
+  input = {
+    id: '',
+    name: '',
+    description: '',
+    title: '',
+  };
+
+  @Input() set data(data: any) {
```

Comment:
you can directly pass the data as `@Input()` without setter. That way it will reduce the amount of the code.once you do that you can get rid of `input`
don't use any. Create an interface.

Reply:
done